codexctl/__init__.py

```diff
@@ -278,6 +316,68 @@ def version_lookup(version: str | None) -> re.Match[str] | None:
 
                 #############################################################
 
+                bootloader_files_for_install = None
+
+                if (device_version_uses_new_engine and
+                    remarkable.hardware == HardwareType.RMPP):
+
+                    current_version = remarkable.get_device_status()[2]
+
+                    if UpdateManager.is_bootloader_boundary_downgrade(current_version, version_number):
+                        print("\n" + "="*60)
+                        print("WARNING: Bootloader Update Required")
+                        print("="*60)
+                        print(f"Current version: {current_version}")
+                        print(f"Target version:  {version_number}")
+                        print()
+                        print("Downgrading from 3.22+ to <3.22 requires updating the")
+                        print("bootloader on both partitions. This process will:")
+                        print("  1. Download the current version's bootloader files")
+                        print("  2. Download the target OS version")
+                        print("  3. Install the target OS version")
+                        print("  4. Update bootloader on both partitions")
+                        print("  5. Reboot")
+                        print()
+
+                        response = input("Do you want to continue? (y/n): ")
```

Comment:
```suggestion
                        response = input("Do you want to continue? (y/N): ")
```
Since N is the default response.